salt/cloud/clouds/vmware.py

```diff
@@ -59,7 +59,7 @@
 # Import python libs
 from __future__ import absolute_import
 from random import randint
```

Comment:
Actually never mind. This test error looks like it is coming from somewhere else as I have seen it in other pull requests.

Thanks @nmadhok!